pkgs/development/compilers/ldc/generic.nix

```diff
@@ -56,16 +56,14 @@ stdenv.mkDerivation rec {
   '';
 
   nativeBuildInputs = [
-    cmake ninja makeWrapper unzip ldcBootstrap lit lit.python
+    cmake ldcBootstrap lit lit.python llvm_11 makeWrapper ninja unzip
   ]
   ++ stdenv.lib.optionals stdenv.hostPlatform.isDarwin [
     darwin.apple_sdk.frameworks.Foundation
     # https://github.com/NixOS/nixpkgs/issues/57120
     # https://github.com/NixOS/nixpkgs/pull/59197#issuecomment-481972515
```

Comment:
What do these comments refer to now? Could probably remove these.

Reply:
Don't think so, those are build dependencies and I didn't saw dependency changes on the release notes of LDC.

Reply:
Those were comments for `llvm_5` so now that `llvm_5` got removed, I think we can remove the comments as well.

Reply:
Resolved, sorry got confused by the diff of the nativeBuildInputs.